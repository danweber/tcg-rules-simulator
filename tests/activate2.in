VERSION:2

TITLE: Activate N of M effects
TEXT: Player chooses, 1 at a time, an effect from a list. No repeats allowed. 

FIELD2:Van,DW1-Van Big,DW1-Big Paper,CS1-02
FIELD1:Green,CS3-02
HAND1:CS3-04
GAME:P1 T3 MAIN 6
DUMP:DP
GET-X-PLAYS

ECHO: ----- play ----
CMD:PLAY CS3-04
STEP:AUTOANSWER:CAPTURE
DUMP:DP