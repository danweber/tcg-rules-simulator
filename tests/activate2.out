FIELD1:Green,CS3-Fairyfeatherlink,1000DP
FIELD2:Van,DW1-Vanillamon,5000DP Big,DW1-Bigmon,12000DP Paper,CS1-02,500DP
GAME:P1 T3 MAIN 6
GET-X-PLAYS: json REFRESH
GET-X-PLAYS: EVOLVE 0 4 3 0 Evolve Lv.3 ArcaneTweet onto FairyFeatherlink (3)
GET-X-PLAYS: PLAY 0 Play ArcaneTweet
GET-X-PLAYS: ATTACK 4 0 Attack FairyFeatherlink into player
GET-X-PLAYS: NEXT PASS TURN
ECHO: ----- play ----
CAPTURE:ArcaneTweet [Effect] 1 of your Monsters gets +2000 DP until the end of your turn.
CAPTURE:ArcaneTweet [Effect] 2 of your opponent's monsters gets -1000 DP until the end of your turn.
CAPTURE:ArcaneTweet [Effect] Delete 1 of your opponent's Monsters with 2000 DP or less.
CAPTURE:Target FairyFeatherlink 4
CAPTURE:Target ArcaneTweet 5
CAPTURE:ArcaneTweet [Effect] 2 of your opponent's monsters gets -1000 DP until the end of your turn.
CAPTURE:ArcaneTweet [Effect] Delete 1 of your opponent's Monsters with 2000 DP or less.
CAPTURE:Target VanillaMon 1
CAPTURE:Target BigMon 2
CAPTURE:Target Paper Shield 3
FIELD1:Green,CS3-Fairyfeatherlink,3000DP Yyy,CS3-Arcanetweet,4000DP
FIELD2:Van,DW1-Vanillamon,4000DP Big,DW1-Bigmon,11000DP Paper,CS1-02,500DP
GAME:P1 T3 MAIN 1
